src/Atc.Azure.Options/ServiceOptions.cs

```diff
@@ -5,10 +5,10 @@ namespace Atc.Azure.Options;
 /// </summary>
 public partial class ServiceOptions
 {
-    public string ApplicationInsightsInstrumentationKey { get; set; } = string.Empty;
```

Comment:
This is a breaking change and it's not urgent as Microsoft will support it until 2025 atleast.
I see two options
- Bump the major version, again, which we quickly can end up doing too often.
- Or as I think we should do, add [Obsolete] attribute to `ApplicationInsightsInstrumentationKey` property and keep supporting it until next major version bump.

Reply:
Yes :)

Reply:
forgot to specify that. We should support both until next major bump and then remove the obsolete property